### PR TITLE
Adds support for random walk component

### PR DESCRIPTION
**Pull request by EiffL**
This  pull request adds support to draw a  GalSim RandomWalk light profile in addition to point sources and   Sersic profiles

### CODE REVIEW

**Comment by EiffL** [comment]
Also, I should note that this requires GalSim v1.5

**Comment by danielsf** [comment]
@EiffL @jchiang87 Is this something that we wanted to attempt in Run 1.1 (I am asking to get a sense of how quickly we need to review this PR)?

**Comment by cwwalter** [comment]
If we can I think yes. Do you agree @jchiang87 and @EiffL ?

**Comment by jchiang87** [comment]
I don't think so, but I could be persuaded otherwise.   This change may be sufficiently involved that we should proceed at a more measured pace. OTOH, I don't know that we have a hard Run1.1i (i for imsim) deadline.  One thing to consider is that if the speckling issue doesn't get resolved quickly, we might want Run1.1i images sooner than we might have expected.

**Comment by EiffL** [comment]
I think the changes are not very extensive and could be implemented in a
1.1 time scale. The question is more what's the ImSim plan for 1.1 (is it
only going to be phosim at this stage ?) And whether imsim is ready for
galsim 1.5

**Comment by danielsf** [comment]
This repo (sims_GalSimInterface) runs against GalSim 1.5.1

I don't think ImSim adds anything on top of this that should pose a problem.

I will look at this PR before the end of the day today.

**Comment by cwwalter** [comment]
imSim has been tested against 1.5.1 and works.   There are a few GalSim deprecation messages that should be taken care of but that is the only issue we know of.

File: python/lsst/sims/GalSimInterface/galSimCatalogs.py [hunk 1]
```diff
@@ -577,6 +577,23 @@ class GalSimGalaxies(GalSimBase, AstrometryGalaxies, EBVmixin):
...
+    like Sersic profiles).
+
+    See the docstring in GalSimBase for explanation of how this class should be used.
+    """
+
+    catalog_type = 'galsim_random_walk'
+    galsim_type = 'RandomWalk'
+    default_columns = [('galacticAv', 0.1, float),
+                       ('galacticRv', 3.1, float),
+                       ('galSimType', 'RandomWalk', str, 9),
```
Comment (danielsf) [inline python/lsst/sims/GalSimInterface/galSimCatalogs.py]: This should be `str, 10` (there are 10 letters in "RandomWalk")

**Comment by danielsf** [comment]
@EiffL  Let me know when you are ready to merge and I will push the big green button (at least, I assume you do not seethe big green button from where you are sitting...)

File: python/lsst/sims/GalSimInterface/galSimCatalogs.py (hunk 1 above)
Comment (EiffL) [reply python/lsst/sims/GalSimInterface/galSimCatalogs.py]: argh, yes, apparently I can't count to 10...

**Comment by EiffL** [comment]
Great, thanks, done. 
I'll also create an issue to remind myself of adding the tests.

**Comment by danielsf** [comment]
@jchiang87 @cwwalter Do I need to cut a new `lsst_sims` right now so that we can start testing ImSim with this?  (There is an outstanding PR in `sims_catUtils` to fix a bug with our AGN model that will need to get folded in before Run 1.1, fyi)

**Comment by jchiang87** [comment]
We can test this work without a lsst_sims release, so I'd wait for the sims_catUtils PR to be resolved.

**Pull request merged**